containers/src/Data/Map/Internal.hs

```diff
@@ -2857,6 +2858,13 @@ filter :: (a -> Bool) -> Map k a -> Map k a
 filter p m
   = filterWithKey (\_ x -> p x) m
 
+-- | \(O(n)\). Filter all keys that satisfy the predicate.
+--
+-- > filterKeys (> 4) (fromList [(5,"a"), (3,"b")]) == singleton 5 "a"
+
```

Comment:
Please add a `@since FIXME` at the end of the docs. It will be updated by maintainers to the next release version.  
Same for IntMap.